src/components/Dashboard/CourtStats.js

```diff
@@ -1,20 +1,131 @@
 import React from 'react'
-import { Box, Distribution } from '@aragon/ui'
+import { Box, GU, textStyle, useTheme } from '@aragon/ui'
+import { formatUnits } from '../../lib/math-utils'
+import useCourtStats from '../../hooks/useCourtStats'
+import {
+  useANJBalanceToUsd,
+  useTokenBalanceToUsd,
+} from '../../hooks/useTokenBalanceToUsd'
+import Loading from './Loading'
+import SplitAmount from '../SplitAmount'
+
+function CourtStats() {
+  const theme = useTheme()
+  const [stats, fetching] = useCourtStats()
 
-const CourtStats = () => {
   return (
-    <Box heading="Court Stats">
-      <Distribution
-        heading="Disputes"
-        items={[
-          { item: 'New', percentage: 37 },
-          { item: 'Adjudicated', percentage: 22 },
-          { item: 'Appeals', percentage: 15 },
-          { item: 'Executed', percentage: 12 },
-        ]}
-      />
+    <Box heading="Court Metrics" padding={3 * GU}>
+      {(() => {
+        if (fetching) {
+          return <Loading height={86} />
+        }
+        return stats.map((stat, index) => {
+          return (
+            <div
+              key={index}
+              css={`
+                margin-bottom: ${2 * GU}px;
+                &:last-child {
+                  margin-bottom: 0;
+                }
+              `}
+            >
+              <span
+                css={`      
+               ${textStyle('body2')}
+               color: ${theme.surfaceContentSecondary};
+               display:block;
+               margin-bottom:${1 * GU}px;
+             `}
```

Comment:
@bpierre mentioned in previos occasions that we should use a semicolon after using `textStyle` so the linter can recognize it and ident the props properly